spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -390,7 +390,7 @@ The implementation must support the following annotation targets:
   * `METHOD`
   * `TYPE`
 
-If the metric no longer exists in the `MetricRegistry` when this annotation is invoked then an `IllegalStateException` will be thrown.
+If the metric no longer exists in the `MetricRegistry` when this annotation element is invoked then an `IllegalStateException` will be thrown.
```

Comment:
annotated element, not annotation element :) https://docs.oracle.com/javase/8/docs/api/java/lang/reflect/AnnotatedElement.html

Reply:
Heh, okay now it's fixed